scripts/check.sh

```diff
@@ -26,10 +26,14 @@ check-protos-compatible() {
 
     if protolock status -lockdir=scripts -protoroot=proto; then
         protolock commit -lockdir=scripts -protoroot=proto
-        return 0
     else
-        return 1
+        echo "Meet break compatibility problem, please check the code."
+        # In order not to block local branch development, when meet break compatibility will force to update `proto.lock`.
```

Comment:
why not the developers use a command to force updates by themselves?

Reply:
Because developers do not know the proto mechanism too well, will be confused to encounter the problem of proto lock update.
And the main role of proto lock is for checking just like a hint,
So force update it in `make check` to decrease developers' learning cost.